packages/tools/devtools/devtools-example/src/App.tsx

```diff
@@ -39,19 +46,46 @@ const sharedTextKey = "shared-text";
  */
 const sharedCounterKey = "shared-counter";
 
+/**
+ * Key in the app's `rootMap` under which the SharedTree object is stored.
+ */
+const sharedTreeKey = "shared-tree";
+
 /**
  * Key in the app's `rootMap` under which the SharedCell object is stored.
  */
 const emojiMatrixKey = "emoji-matrix";
 
+
+/**
+ * SharedTree class object containing static factory method used for {@link @fluidframework/fluid-static#IFluidContainer}.
+ */
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+class SharedTree {
```

Comment:
nit: you can define this class inside the body of the `castSharedTreeType` function. Then, the class is not accessible anywhere else and the `castSharedTreeType` function entirely contains the kludge. Bonus: then you don't need to document it ;) (although you should still document the `castSharedTreeType` function so other people know why we need it for now).

Reply:
🐐 feedback. Thank you!